openshift/ci-operator/generate-ci-config.sh

```diff
@@ -41,6 +41,20 @@ EOF
   done
 }
 
+function generate_cron_expression {
+  if [[ "$branch" == "knative-v0.25.3" ]]; then
+    echo '0 1 * * 1-5'
+  elif [[ "$branch" == "knative-v0.26" ]]; then
+    echo '0 3 * * 1-5'
+  elif [[ "$branch" == "knative-v1.0" ]]; then
+    echo '0 5 * * 1-5'
+  elif [[ "$branch" == "knative-v1.1" ]]; then
+    echo '0 7 * * 1-5'
+  elif [[ "$branch" == "knative-v1.2" ]]; then
+    echo '0 9 * * 1-5'
+  fi
+}
```

Comment:
I'm wondering if we can avoid hard-coding these branches here? :thinking:

Reply:
yeah, I know what you mean. Alternative is maybe create a file in each branch that has the Cron expression. Then we can read the file in the script.

I thought about that but this hardcoding is at least making things more central and I don't have to look at 5 different branches for the cron expression when I want to see when jobs are running.

I am ok with both if you have a strong opinion.

Reply:
Or, any other ideas?

Reply:
I'm fine with starting with this to validate the idea and not spend much effort in crappy shell until it's proven, then move to a more dynamic random distribution using modulo 
https://stackoverflow.com/questions/13104478/uniformity-of-random-numbers-taken-modulo-n

Reply:
https://github.com/openshift/knative-eventing/pull/1639#discussion_r807849476